### PR TITLE
Filecoin Objectives, Summary and Quiz

### PR DESCRIPTION
**Pull request by walkerlj0**
Added an Objectives page with a basic description, and improved the Introduction page

### CODE REVIEW

File: content/en/curriculum/filecoin/introduction/index.md [hunk 1]
```diff
@@ -14,16 +14,20 @@ level:
 
 # Filecoin
 
-**Filecoin in the incentivation structure** for IPFS.
+Filecoin is an independent organization that was founded by, and now collaborates with Protocol Labs and the PLN, and is the token behind decentralized storage. Filecoin uses IPFS, libp2p, IPLD, and many other technologies to create protocols, tools, and services to help radically improve the and drive breakthroughs in computing.
```
Comment (mrodriguez3313) [inline content/en/curriculum/filecoin/introduction/index.md:17]: > and services to help radically improve the and drive breakthroughs in computing.

I think this is supposed to say:

```suggestion
Filecoin is an independent organization that was founded by, and now collaborates with Protocol Labs and the PLN, and is the token behind decentralized storage. Filecoin uses IPFS, libp2p, IPLD, and many other technologies to create protocols, tools, and services to help radically improve and drive breakthroughs in computing.
```

Comment (mrodriguez3313) [reply content/en/curriculum/filecoin/introduction/index.md:17]: or Improve the network and drive breakthroughs

**Review by mrodriguez3313 — APPROVED** [review]
@walkerlj0 Just the one comment. I think you would know better of what you were going for there.

File: content/en/curriculum/filecoin/introduction/index.md [hunk 2]
```diff
@@ -14,16 +14,20 @@ level:
 
 # Filecoin
 
-**Filecoin in the incentivation structure** for IPFS.
+Filecoin is an independent organization that was founded by, and now collaborates with Protocol Labs and the PLN, and is the token behind decentralized storage. Filecoin uses IPFS, libp2p, IPLD, and many other technologies to create protocols, tools, and services to help radically improve and drive breakthroughs in computing.
```
Comment (walkerlj0) [inline content/en/curriculum/filecoin/introduction/index.md:17]:
```suggestion
Filecoin is an independent organization that was founded by, and now collaborates with Protocol Labs and the PLN, and is the token behind decentralized storage. Filecoin uses IPFS, libp2p, IPLD, and many other technologies to create protocols, tools, and services to help radically improve the network and drive breakthroughs in computing.
```

**Pull request merged**